bs/src/tablecloth.mli

```diff
@@ -1527,41 +1527,345 @@ module Result : sig
     -> unit
 end
 
+(**
+  This module implements the `Option` type, which has a variant for 
+  valid values (`'Some`), and one for invalid values (`'None`).
```

Comment:
I think it's more accurate to say it provides functions to work on the option type. While the `t` exists, I don't think anyone is going to use `Option.t` instead of `option` (nor should they).

Reply:
Sounds good to me.